client/components/admin/edit-song.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable react/no-unused-state */
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+//i think we need a thunk creator to edit song also?
+//updateSong currently does not exist
+import {updateSong} from '../../store/songs'
```

Comment:
yes, I do think we need a thunk creator that dispatches a .put route request from our Songs Api file